### PR TITLE
install tiny invariant

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

Add invariant package: https://www.npmjs.com/package/tiny-invariant for the next PR in the stack
fixes: #1728
## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1730** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1730" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1730?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1730?utm_source=stack-comment).


Job Reference: JoerlrxKBtAdcxqCKXjb

**Pull request merged**
